aspnetcore/tutorials/web-api-help-pages-using-swagger.md

```diff
@@ -110,239 +80,217 @@ In Visual Studio, press ^F5 to launch the app and navigate to `http://localhost:
 
-Notice the enhancement of the UI with these additional comments.
+Notice the UI enhancements with these additional comments.
 
 ![Swagger UI with additional comments shown](web-api-help-pages-using-swagger/_static/xml-comments-extended.png)
 
 ### DataAnnotations
 
 You can decorate the API controller with `System.ComponentModel.DataAnnotations` to help drive the Swagger UI components.
 
-Adding the `[Required]` annotation to the `Name` property of the `TodoItem` class will change the ModelSchema information in the UI. `[Produces("application/json")]`, `RegularExpression` validators and more will further detail the information delivered in the generated page.  The more metadata that is in the code produces a more desciptive UI or API help page.  
+Adding the `[Required]` annotation to the `Name` property of the `TodoItem` class changes the UI behavior and alters the underlying JSON schema:
+
+```json
+"definitions": {
+    "TodoItem": {
+        "required": [
+            "name"
+        ],
+        "type": "object",
+        "properties": {
+            "id": {
+                "format": "int64",
+                "type": "integer"
+            },
+            "name": {
+                "type": "string"
+            },
+            "isComplete": {
+                "default": false,
+                "type": "boolean"
+            }
+        }
+    }
+},
+```
 
-[!code-csharp[Main](../tutorials/web-api-help-pages-using-swagger/sample/src/TodoApi/Models/TodoItem.cs?highlight=10)]
+`[Produces("application/json")]`, `RegularExpression` validators, and more, further detail the information delivered in the generated page. The more metadata that is in the code, the more descriptive the UI or API help page becomes.
 
+[!code-csharp[Main](../tutorials/web-api-help-pages-using-swagger/sample/TodoApi/Models/TodoItem.cs?highlight=10)]
 
 ### Describing Response Types
 
-Consuming developers are probably most concerned with what is returned; specifically response types, error codes (if not standard). These are handled in the XML comments and DataAnnotations.
+Consuming developers are most concerned with what is returned &mdash; specifically response types and error codes (if not standard). These are handled in the XML comments and data annotations.
 
-Take the `Create()` method for example, currently it returns only "201 Created" response by default. That is of course if the item is in fact created, or a "204 No Content" if no data is passed in the POST Body.  However, there is no documentation to know that or any other response. That can be fixed by adding the following piece of code.
+Take the `Create` method for example. It returns only a "201 Created" response, by default. That is, of course, if the item is in fact created, or a "204 No Content" if no data is passed in the POST Body. However, there is no documentation to know that or any other response. That can be fixed by adding the following piece of code:
```

Comment:
Rewrite this and remove dead wood. SOmething like
The `Create` method returns `201 created`  on success or "204 No Content" on failure. There is no doc ...